run.py

```diff
@@ -576,7 +575,12 @@ def run(args):
         suite_file_name = " ".join(suite_file_name.split("_"))
         _log = run_dir.replace("/ceph/", "http://magna002.ceph.redhat.com/")
 
-        launch_name = f"RHCS {rhbuild} - {suite_file_name}"
+        launch_suffix = f"{rhbuild} - {suite_file_name}"
+        launch_name = (
+            f"Upstream {launch_suffix}"
```

Comment:
I think `upstream` should not have `rhbuild` in suffix.

Reply:
Made changes